packages/app-backend-vue2/src/components/tree.ts

```diff
@@ -191,7 +194,7 @@ async function capture (instance, index?: number, list?: any[]): Promise<Compone
       inactive: false,
       isFragment: false // TODO: Check what is it for.
     }
-    return api.visitComponentTree(
+    return await api.visitComponentTree(
```

Comment:
Await in front of return is unnecessary

Reply:
Resolved in 93f94fda5dc5a780c53fe57112f2bcdb9b1360f6